src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -422,9 +422,10 @@
   (on-long-press
    (concat
     (when (and outgoing edit-enabled)
-      [{:on-press #(re-frame/dispatch [:chat.ui/edit-message message])
-        :label    (i18n/label :t/edit)
-        :id       :edit}])
+      (let [mentionable-users @(re-frame/subscribe [:chats/mentionable-users])]
+        [{:on-press #(re-frame/dispatch [:chat.ui/edit-message message mentionable-users])
+          :label    (i18n/label :t/edit)
+          :id       :edit}]))
```

Comment:
This looks a little weird, I am not sure if it is the right approach to use subscribe here.
Should we avoid using subscribe all along here, and construct the mentionable-users object by directly calling to function?
https://github.com/status-im/status-mobile/blob/99b28095327abc89df7cfd5bef59e71a4205a69b/src/status_im/subs/chat/chats.cljs#L397

@flexsurfer, wdyt?